### PR TITLE
Fix/download file bug

### PR DESCRIPTION
**Pull request by DanielZhangReal**
- Fix `private-transfer.lfs` file download failure in Manta Wallet
- Add HttpProvider
- Fix the error that the download failed on Ledger api  @zqhxuyuan 

---

Before we can merge this PR, please make sure that all the following items have been checked off:

- [x] Linked to an issue with discussion and accepted design OR have an explanation in the PR that describes this work.
- [x] Added **one** line describing your change in [`CHANGELOG.md`](https://github.com/manta-network/sdk/blob/main/CHANGELOG.md) and added the appropriate `L-` label to the PR.
- [x] Re-reviewed `Files changed` in the GitHub PR explorer.

### CODE REVIEW

**Review by Apokalip — DISMISSED** [review]

File: manta-js/examples/extension-example/src/App.tsx [hunk 1]
```diff
@@ -55,7 +61,11 @@ export default function App() {
     setZkAddress(accounts[0].zkAddress);
     setInjected(injected);
 
-    const provider = new WsProvider(rpcUrl);
+    // const provider = new WsProvider(rpcUrl);
+    const provider = new HttpProvider(
+      rpcUrl[(Math.random() * rpcUrl.length) | 0],
```
Comment (ferrell-code) [inline manta-js/examples/extension-example/src/App.tsx:66]: I believe you need to do `Math.floor(Math.random() % rpcUrl.length)` or you are accessing list using floats

**Review by ferrell-code — COMMENTED** [review]
One comment looks good

File: manta-js/examples/extension-example/src/App.tsx (hunk 1 above)
Comment (DanielZhangReal) [reply manta-js/examples/extension-example/src/App.tsx:66]: `(Math.random() * 3) | 0`

`| 0`

Comment (ferrell-code) [reply manta-js/examples/extension-example/src/App.tsx:66]: Ah gotcha the `| 0` truncates the value to integer

**Review by ferrell-code — APPROVED** [review]

**Review by Apokalip — APPROVED** [review]

**Pull request merged**